.github/workflows/dapr-ambient.yaml

```diff
@@ -0,0 +1,249 @@
+        target_os: ["linux"]
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Parse release version and set REL_VERSION and LATEST_RELEASE
+        run: python ./.github/scripts/get_release_version.py ${{ github.event_name }}
+
+      - name: Set REPO_OWNER
+        shell: bash
+        run: |
+          REPO_OWNER=${{ github.repository_owner }}
+          # Lowercase the value
+          echo "REPO_OWNER=${REPO_OWNER,,}" >>${GITHUB_ENV}
+
+      - name: GitHub Container Registry login
+        uses: docker/login-action@v2
+        if: matrix.target_os != 'darwin'
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+  helm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      
+      - name: Parse release version and set REL_VERSION and LATEST_RELEASE
+        run: python ./.github/scripts/get_release_version.py ${{ github.event_name }}
+      
+      - name: setup helm
+        run: |
+          curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+          sudo apt-get install apt-transport-https --yes
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+          sudo apt-get update
+          sudo apt-get install helm
+
+      - name: install helm
+        run: helm plugin install https://github.com/quintush/helm-unittest
+      
+      - name: execute unit tests
+        run: helm unittest chart/dapr-ambient
+
+      - name: update docs
+        if: github.ref == 'refs/heads/main'
+        run: |
+          wget https://github.com/norwoodj/helm-docs/releases/download/v1.11.0/helm-docs_1.11.0_Linux_x86_64.tar.gz
+          tar -zxf helm-docs_1.11.0_Linux_x86_64.tar.gz
+          ./helm-docs
+
+      - name: commit and push changes
+        if: github.ref == 'refs/heads/main'
+        env:
+          COMMIT_MESSAGE: "docs: update README.md"
+          COMMIT_AUTHOR: dapr-ambient-bot
+        run: |
+          mv chart/dapr-ambient/README.md README.md
+          git config --global user.name "${{ env.COMMIT_AUTHOR }}"
+          git config --global user.email "dapr-ambient-bot@diagrid.com"
+          git add README.md
+          git commit -m "${{ env.COMMIT_MESSAGE }}"
+          git push origin ${{ github.head_ref || github.ref_name }}
+      
+      # NEEDS MODIFICATION: PUSH TO AZURE ECR
+      - name: Package & Push Helm Chart
+        env:
+            ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        run : |
+          helm package \
+          ./chart/dapr-ambient \
+          --version ${RELEASE_VERSION}
+          helm push "${RELEASE_NAME}-${RELEASE_VERSION}.tgz" oci://${ECR_REGISTRY}
```

Comment:
@asarenkansah I think we just need to use ${{ secrets.DOCKER_REGISTRY }} there instead of ${ECR_REGISTRY}

Can we try that?